### ISSUE TITLE
[Question] Is it possible to use LightGBMModel with early_stopping

### ISSUE THREAD

**Issue by netojoseaugusto** [unit8co/darts#1593]
Hello, 

This is more of a question than a feature request. To use early_stopping with lgbm one has to provide a validation set from which one metric will be calculated and, if this metric doesn't improve for a given number of rounds, the training is stopped. I would like to know if it's somehow possible to do the same using LightGBMModel or if exists any workaround to use early_stopping with LightGBMModel.

Thank you

**Comment by madtoinou** [comment]
Hi,

Yes it's possible to use early stopping with the LGBMModel using the `val_series` argument ([documentation](https://unit8co.github.io/darts/generated_api/darts.models.forecasting.lgbm.html?highlight=lgbm#darts.models.forecasting.lgbm.LightGBMModel.fit)) and the `early_stopping_round` kwarg ([documentation](https://lightgbm.readthedocs.io/en/latest/Parameters.html)) in the `fit()` method.

**Comment by netojoseaugusto** [comment]
Hi,

Thanks very much, @madtoinou. Really don't know how I missed that since it was so clear in the docs.
